src/libraries/Common/src/Interop/Unix/System.Native/Interop.LChflags.cs

```diff
@@ -22,5 +22,11 @@ internal enum UserFlags : uint
         [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_LChflagsCanSetHiddenFlag")]
         [SuppressGCTransition]
         private static partial int LChflagsCanSetHiddenFlag();
+
+        internal static readonly bool SupportsHiddenFlag = (CanGetHiddenFlag() != 0);
+
+        [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_CanGetHiddenFlag")]
+        [SuppressGCTransition]
+        private static partial int CanGetHiddenFlag();
```

Comment:
would it be enough to just change the return type to `bool`?

```suggestion
        internal static readonly bool SupportsHiddenFlag = CanGetHiddenFlag();

        [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_CanGetHiddenFlag")]
        [SuppressGCTransition]
        private static partial bool CanGetHiddenFlag();
```

Reply:
For consistency, I used the same return type as `LChflagsCanSetHiddenFlag`.